test/mtz-wizard_test.html

```diff
@@ -224,11 +224,15 @@
             });
 
             it('should return false if any step fails validation', () => {
-              expect(element.validate(steps)).to.equal(false);
+              element.validate(steps).then((isValid) => {
```

Comment:
Return the promise chain on each of these

Reply:
If you return a promise in a test it will wait for the promise to resolve before it finishes the test, more optimal to using `done()` but the same principles.

```
return element.validate(steps).then((isValid) => {
  expect(isValid).to.be.true;
});
```

Reply:
When you say return the promise chain, I don't want validate to return a promise chain because that'd break backward compatibility. What specifically do you mean by return the promise chain here?

Reply:
The validate function in each step needs to return a promise instead of a Boolean, then the validate function in wizard needs to return a promise instead of a boolean, that's the only way we can support what's needed in promo-builder.  It's a breaking change but it's one we need, and it's a 0.x so we can do that with a minor bump, not a big deal.  Once that's done we need to update this test as above so it's consuming a promise, will turn this test into an async test, similar to how calling done() works except it's bound to the promise chain resolving so you don't have to use the callback.